src/common/mod.rs

```diff
@@ -232,6 +232,8 @@ pub enum Method {
     Put,
     /// PATCH Method.
     Patch,
+    /// Delete Method.
+ 	Delete,
```

Comment:
Use space instead of tab？

Reply:
Add an extra tab here. :( Sorry for that